hugegraph-hbase/pom.xml

```diff
@@ -34,14 +34,90 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.baidu.hugegraph</groupId>
-            <artifactId>hbase-shaded-endpoint</artifactId>
-            <version>2.0.6</version>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-endpoint</artifactId>
+            <version>3.0.0-alpha-3</version>
```

Comment:
seems a little danger? the biggest version in `2.x` fail?

Reply:
yes, 2.x version use protobuf 2, conflict